PayPalNativePayments/src/main/java/com/paypal/android/paypalnativepayments/PayPalNativeCheckoutClient.kt

```diff
@@ -163,7 +160,7 @@ class PayPalNativeCheckoutClient internal constructor(
                 ShippingChangeType.OPTION_CHANGE -> {
                     analyticsService.sendAnalyticsEvent(
                         "paypal-native-payments:shipping-method-changed",
-                        orderId
+                        null
```

Comment:
Order ID not available in MXO shipping callback.

Reply:
I _think_ the `payToken` in `ShippingChangeData` may be an order ID? But maybe @saperi22 or @tdchow can confirm.